src/main/java/io/rpg/wrapper/ConfigChooser.java

```diff
@@ -0,0 +1,34 @@
+package io.rpg.wrapper;
+
+import java.io.File;
+import java.util.Optional;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+
+public class ConfigChooser {
+  private final FileChooser chooser;
+
+  public ConfigChooser() {
+    chooser = new FileChooser();
+    chooser.setTitle("Select root.json");
+    chooser.getExtensionFilters().addAll(
+        new FileChooser.ExtensionFilter("config", "root.json"),
+        new FileChooser.ExtensionFilter("json", "*.json")
+    );
+  }
```

Comment:
I believe `ConfigLoader` actually expects the path to the directory that contains configuration (including `root.json`), not to `root.json` file.